### DISCUSSION TITLE
Docker entrypoint changed in 2.0.0-12 (breaking change)

### DISCUSSION THREAD

**Discussion by bmerigan** [artilleryio/artillery#1350] [Artillery]
Just raising awareness that there is a breaking change in 2.0.0-12
File: `Dockerfile`
Old code:
`ENTRYPOINT ["/home/node/artillery/bin/artillery"]`
New code:
`ENTRYPOINT ["/home/node/artillery/bin/run"]`

We were using `:latest` and this broke our build this morning.

**Comment by hassy** [comment]
We didn't mention it in the release notes as we don't expect users to rely on the specific entrypoint in the image, but perhaps we should. What did that change break for you @bmerigan?

thanks for letting us know!

**Comment by bmerigan** [comment]
In our Dockerfile we specifically `chmod` that directory to be executable (which is not our default).  When the path disappeared our build failed.
